README.md

```diff
@@ -1,15 +1,17 @@
 # stactools-modis
 
-Template repostitory for [stactools](https://github.com/stac-utils/stactools) packages.
-
-## How to use
-
-1. Clone this repository and name it `stactools-{NAME}`, where `NAME` is your package name.
-   This name should be short, memorable, and a valid Python package name (i.e. it shouldn't start with a number, etc).
-2. Update `setup.cfg` with your package name, description, and such.
-3. Rename `src/stactools.modis` to `src/stactools/{NAME}`.
-4. Rewrite this README to provide information about how to use your package.
-5. Update the LICENSE with your company's information (or whomever holds the copyright).
-6. Update the environment name in `environment.yml`.
-7. Update the environment variables in `.github/workflows/release.yml` to the appropriate values to publish for your organization.
-8. Update all scripts in the `docker` directory to refer to `stactools-{NAME}` and `stactools-{NAME}-dev`.
+- Name: modis
+- Package: 'stactools.modis'
+- PyPI: https://pypi.org/project/stactools-modis/
+- Owner:
+- Dataset homepage: 
+- STAC extensions used:
+- Extra Fields: 
+
+This repository will assist you in the generation of STAC files for MODIS datasets. 
+
+## Examples
+
+### STAC Objects
+
+### Command-line Usage
```

Comment:
To create a STAC `Item`:

```shell
$ stac modis create-item tests/data-files/MCD12Q1.A2001001.h00v08.006.2018142182903.hdf.xml build
```

To create a STAC `Catalog` from a list of MODIS asset hrefs:

```shell
$ stac modis create-catalog examples/file-list.txt examples
```

Note that this `create-catalog` example is exactly how the `examples/` directory is generated.